src/app/static/src/app/components/plots/ResetMap.vue

```diff
@@ -6,7 +6,9 @@
                 href="#" :title="tooltipContent('resetView')" 
                 role="button" 
                 :aria-label="tooltipContent('resetView')">
-                <vue-feather type="refresh-cw" size="20"></vue-feather>
+                <vue-feather type="refresh-cw"
+                             size="20"
+                             style="vertical-align: middle; margin-bottom: 3px; margin-right: 1px;"></vue-feather>
```

Comment:
even centering the reset map icon normally is slightly off, might be to do with vue feather so need a manual amount of centering